nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -189,34 +223,58 @@ define([
             // widgets this cell owns
             cellTabs,
             controlPanel,
-            fileTypePanel,
-            model = Props.make({
-                data: TestAppObj,
-                onUpdate: function (props) {
-                    Utils.setMeta(this.cell, 'bulkImportCell', props.getRawObject());
-                },
-            });
+            fileTypePanel;
         if (options.initialize) {
-            initialize(typesToFiles);
+            initialize(options.specs);
         }
+        let model = Props.make({
```

Comment:
Now, instead of running `Props.make` with the test data, this can use the actual data from the cell.